tutorials/tour/anonymous-function-syntax.md

```diff
@@ -51,3 +51,33 @@ Function1[Int, Int]
 Function2[Int, Int, String]
 Function0[String]
 ```
+
+The following example shows how to use anonymous function of the beginning of this page
+
+```tut
+package tour
+
+object AnonymousFunction {
+
+  /**
+   * Method to increment an integer by one.
+   */
+  var anonymousIncrementFunction = (x: Int) => x + 1
+
+  /**
+   * Main method
+   * @param args application arguments
+   */
+  def main(args: Array[String]) {
+
+    // Create an integer to test the anonymous function with
+    var myInteger: Int = 0;
+
+    println(myInteger) // Prints: 0
+
+    myInteger = anonymousIncrementFunction(myInteger)
```

Comment:
I couldn't ommit the var in the two places it appeared, else the code did not compile. I guess it is 'truly needed' in those cases.